lib/services/northBound/deviceGroupAdministrationServer.js

```diff
@@ -306,6 +324,34 @@ function clear(callback) {
     callback();
 }
 
+function renameObjetKeys(keysMap, obj) {
+    return Object.keys(obj).reduce((acc, key) => {
+        // Si la clave existe en el mapa, usar la clave renombrada
+        const renamedKey = keysMap[key] || key;
+
+        // Agregar la clave renombrada al acumulador
+        acc[renamedKey] = obj[key];
+
+        return acc;
+    }, {});
+}
+
+function checkAndModifyGroupResp(route, payload) {
+    if (route === '/iot/services') {
+        return payload;
+    } else {
+        return renameObjetKeys({ services: 'groups' }, payload);
```

Comment:
maybe you can address (payload.groups=services); delete payload.services explicitly here

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1648/commits/936d78cc08e107b11275866519e221f88f134b6f